docs/docs/integrations/css-prop.md

```diff
@@ -27,7 +27,14 @@ List the plugin in `.babelrc`:
 Usage:
```

Comment:
This file had a conflict, so when the pre-commit formatting run happened, it was included. No changes to this file that are important.